src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleEngine.java

```diff
@@ -268,13 +274,51 @@ protected void setTransactionIsolation() throws SQLException {
     @Override
     protected void createTable(final DbEntity entity) throws DatabaseEngineException {
 
-        List<String> createTable = new ArrayList<>();
+        // Get default create table statement
+        final String defaultCreateTableStatement = getCreateTableStatement(entity);
+
+        // Check for extra properties
+        final String createTableCompressLOBS = getCompressLobsStatements(entity);
+
+        final String createTableStatement = defaultCreateTableStatement + createTableCompressLOBS;
+
+        for (final String createTable : ImmutableList.of(createTableStatement, defaultCreateTableStatement)) {
+            try (
+                    Statement statement = conn.createStatement()
+            ) {
+                statement.executeUpdate(createTable);
+                break; // If we got here, then we successfully created the table
+            } catch (final SQLException ex) {
+                if (ex.getMessage().startsWith(NAME_ALREADY_EXISTS)) {
+                    logger.debug(dev, "'{}' is already defined", entity.getName());
+                    handleOperation(new OperationFault(entity.getName(), OperationFault.Type.TABLE_ALREADY_EXISTS), ex);
+                    break; // Name already exists, we cannot do anything so we exit
```

Comment:
nothing to do after this... could be a _return;_ instead?

Reply:
It was meant to be a return: somehow I put a break there (facepalm).